custom_components/ha-google-home/utils.py

```diff
@@ -16,19 +16,21 @@ def convert_from_ms_to_s(timestamp):
     return round(timestamp / 1000)
 
 
-def format_timer_information(timer):
-    timer[FIRE_TIME_IN_S] = convert_from_ms_to_s(timer[FIRE_TIME])
+def format_timer_information(timer_timestamp):
```

Comment:
`timer_timestamp` is very confusing name. This is a whole dict with timer attributes. I'd keep old name.